scylla-cql/src/serialize/value_tests.rs

```diff
@@ -1635,6 +1635,24 @@ fn text_serialization() {
     );
 }
 
+#[test]
+fn box_str_serialization() {
+    let val: Box<str> = "abc".into();
+    assert_eq!(
+        do_serialize(val, &ColumnType::Native(NativeType::Text)),
+        vec![0, 0, 0, 3, 97, 98, 99]
+    );
+}
+
+#[test]
+fn arc_str_serialization() {
+    let val: Arc<str> = "abc".into();
+    assert_eq!(
+        do_serialize(val, &ColumnType::Native(NativeType::Text)),
+        vec![0, 0, 0, 3, 97, 98, 99]
+    );
+}
+
```

Comment:
Could you move those cases into `text_serialization`? This test already covers both str and String, so I think it can also cover Box and Arc.